rewrite-java-21/src/main/java/org/openrewrite/java/isolated/internal/Javac.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2009-2019 The Project Lombok Authors.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.openrewrite.java.isolated.internal;
+
+
+import org.openrewrite.java.isolated.internal.JavacTreeMaker.TypeTag;
+
+import static org.openrewrite.java.isolated.internal.JavacTreeMaker.TypeTag.typeTag;
+import static org.openrewrite.java.isolated.internal.JavacTreeMaker.TypeTag.typeTagPermissive;
+
+public final class Javac {
+    private Javac() {
+        // prevent instantiation
+    }
```

Comment:
```suggestion
import lombok.experimental.UtilityClass;

@UtilityClass
public final class Javac {
```

Reply:
So these files are copied from Lombok; I wouldn't spend too much time maintaining these, as they are likely to be dropped when we do get around to rewrite-java-25. Until then they can stay as is. See also:
- https://github.com/openrewrite/rewrite/pull/5309/files/d39c4f226cbba2243666831712b76a60899929a3#r2047439216